modules/installation-azure-marketplace-subscribe.adoc

```diff
@@ -129,7 +129,47 @@ ifdef::ipi[]
 . Record the image details of your offer. You must update the `compute` section in the `install-config.yaml` file with values for `publisher`, `offer`, `sku`, and `version` before deploying the cluster.
 endif::ipi[]
 ifdef::upi[]
-. Record the image details of your offer. If you use the Azure Resource Manager (ARM) template to deploy your worker nodes, you can update `storageProfile.imageReference` by deleting the `id` parameter and adding the `offer`, `publisher`, `sku`, and `version` parameters using the values from your offer.
+. Record the image details of your offer. If you use the Azure Resource Manager (ARM) template to deploy your worker nodes, you can update `storageProfile.imageReference` by deleting the `id` parameter and adding the `offer`, `publisher`, `sku`, and `version` parameters by using the values from your offer.
++
+.Sample `06_workers.json` ARM template with an updated `storageProfile.imageReference` object
++
+[source,json]
+----
+...
+  "storageProfile": {
+    "imageReference": {
+      "offer": "rh-ocp-worker",
+      "publisher": "redhat",
+      "sku": "rh-ocp-worker",
+      "version": "4.8.2021122100"
+  }
+}
+...
+----
+
+. Add a plan to the ARM template for your worker nodes.
++
+[IMPORTANT]
```

Comment:
While true, I suggest removing this admonition. You are explicitly telling users to complete a required step. No need to tell them what happens if they do not follow the doc. ;)

Reply:
Yeah, good point. Removed. 

I misunderstood that the added step was somewhat assumed based on how it was missing from the original steps.